packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -106,7 +106,11 @@ const propTypes = {
   onDisableSelectableRows: PropTypes.func,
 
   /**
-   * Boolean indicating whether or not the DataGrid should allow entire rows to be selected. An additional column will be
+   * Callback function to enable row selection mode and select a given set of rows. Parameters: `function(arrayOfRowIds)`
+   */
+  onEnableSelectableRows: PropTypes.func,
```

Comment:
Should this be a Boolean prop instead of a callback function?  I would think that onRowSelect would be updated to return an array of rows.  We don't want multiple options for returning the selected items.